front/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "front",
```

Comment:
change project name to something that describes it better, lms-ui, for example

Reply:
done